tests/js/app/testArticleHTMLRenderer.js

```diff
@@ -74,8 +74,8 @@ describe('Article HTML Renderer', function () {
         let html_no_title = renderer.render(wikibooks_model);
         renderer.show_title = true;
         let html_with_title = renderer.render(wikibooks_model);
-        expect(html_with_title).toMatch('Wikibooks title');
-        expect(html_no_title).not.toMatch('Wikibooks title');
+        expect(html_with_title).toMatch('<h1>Wikibooks title</h1>');
```

Comment:
We now inject the metadata into the article, this means that the title string will always be present in the content, therefore, this test needs to differentiate between "just matching the string" vs "matching the string when is presented as the title", that is why adding that presentation detail is necessary.